frame/staking/src/lib.rs

```diff
@@ -1427,7 +1478,13 @@ decl_module! {
 				ledger = ledger.consolidate_unlocked(current_era)
 			}
 
-			let post_info_weight = if ledger.unlocking.is_empty() && ledger.active < T::Currency::minimum_balance() {
+			let is_validator = Validators::<T>::contains_key(&stash);
```

Comment:
I think we should prevent this when we call `unbond`, not here. In essence, a validator can now have less than `MinValidatorBond ` bonded, but they can't withdraw it. We probably want at least `MinValidatorBond ` to be _bonded_. 

This can stay with `ED`.

Reply:
Not addressed ⚠️